dashboard/app/controllers/plc/enrollment_evaluations_controller.rb

```diff
@@ -2,6 +2,9 @@ class Plc::EnrollmentEvaluationsController < ApplicationController
   before_action :load_and_authorize_for_this_unit
 
   def preview_assignments
+    modules_to_enroll_in = Plc::LearningModule.find(@enrollment_unit_assignment.plc_course_unit.determine_preferred_learning_modules(current_user))
```

Comment:
Accessing `current_user` in the model probably isn't a good idea.  I think it's fine to explicitly pass it in though (@wjordan may have a different opinion).